frameworks/PHP/php-ngx/deploy/nginx_plain_php.conf

```diff
@@ -35,7 +35,7 @@ http {
         root /;
         index  index.html;
 
-        php_keepalive 100;
+        php_keepalive 200;
```

Comment:
Add that change to deploy/nginx.conf too.

Reply:
> Add that change to `deploy/nginx.conf` too.

@joanhey Fixed it by https://github.com/TechEmpower/FrameworkBenchmarks/pull/5138/commits/c78769acd678172401ceab26b9b3b1e066019c84